repository-data/application/src/main/resources/hcm-content/taxonomies.yaml

```diff
@@ -0,0 +1,2836 @@
+          hippotaxonomy:key: neurology
+          locale.en.name: Neurology
+        /neurosurgery:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: neurosurgery
+          locale.en.name: Neurosurgery
+        /nuclear_medicine:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: nuclear_medicine
+          locale.en.name: Nuclear medicine
+        /obstetrics_and_gynaecology:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: obstetrics_and_gynaecology
+          locale.en.name: Obstetrics and gynaecology
+        /occupational_medicine:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: occupational_medicine
+          locale.en.name: Occupational medicine
+        /old_age_psychiatry:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: old_age_psychiatry
+          locale.en.name: Old age psychiatry
+        /ophthalmology:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: ophthalmology
+          locale.en.name: Ophthalmology
+        /oral_and_maxillofacial_surgery:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: oral_and_maxillofacial_surgery
+          locale.en.name: Oral and maxillofacial surgery
+        /otorhinolaryngology__ear,_nose_and_throat_surgery,_ent_:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: otorhinolaryngology__ear,_nose_and_throat_surgery,_ent_
+          locale.en.name: Otorhinolaryngology (ear, nose and throat surgery, ENT)
+        /paediatric_cardiology:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: paediatric_cardiology
+          locale.en.name: Paediatric cardiology
+        /paediatric_surgery:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: paediatric_surgery
+          locale.en.name: Paediatric surgery
+        /paediatrics:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: paediatrics
+          locale.en.name: Paediatrics
+        /palliative_medicine:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: palliative_medicine
+          locale.en.name: Palliative medicine
+        /pharmaceutical_medicine:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: pharmaceutical_medicine
+          locale.en.name: Pharmaceutical medicine
+        /plastic_surgery:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: plastic_surgery
+          locale.en.name: Plastic surgery
+        /psychiatry_of_intellectual_disability__pid_:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: psychiatry_of_intellectual_disability__pid_
+          locale.en.name: Psychiatry of intellectual disability (PID)
+        /public_health__doctor_:
```

Comment:
Can we remove the trailing `_` from the name and key ?

Reply:
Going through and changing all of these now